manim/cli/project/templates/default.py

```diff
@@ -0,0 +1,12 @@
+class default(Scene):
```

Comment:
This name doesn't conform to [PEP8](https://www.python.org/dev/peps/pep-0008/#class-names)

Reply:
thanks for review i will change it